deps/.gitignore

```diff
@@ -0,0 +1,29 @@
+!/v8/test/debugger/debug/
+!/v8/src/base/debug
+!/v8/src/debug
+/v8/src/debug/obj
+
+!/v8/tools/release
+
+/v8-*
+/icu
+/icu*.zip
+/icu*.tgz
+/icu-tmp
+
+# generated by gyp on Windows
+/openssl/openssl.props
+/openssl/openssl.targets
+/openssl/openssl.xml
+
+!/npm/node_modules**
```

Comment:
should this be !/npm?

Reply:
the other files in `/deps/npm` aren't ignores by any other rule, only `node_modules` is. So this is the minimal glob needed.